submissions/smillims/html-css-popup/css/media.css

```diff
@@ -0,0 +1,48 @@
+@media screen and (max-width: 655px) {
```

Comment:
It's not a blocker. But it seems something happened with the formatting of this file. 
I recommend using Prettier for the formatting of your code.